src/imitation/util/logger.py

```diff
@@ -125,6 +125,9 @@ def get_dir(self) -> str:
     def log(self, *args, **kwargs):
         self.default_logger.log(*args, **kwargs)
 
+    def warn(self, *args):
```

Comment:
Huh? Is this necessary? I think `warn` in the super class should call `self.log` which will then do the appropriate thing of forwarding to `self.default_logger`.

Note if we did want to add `warn`, we'd need to also add `info`, `error`, etc.

Reply:
Oh yeah, you're completely right